robot/basestation/static/js/elements/navigation/localization.js

```diff
@@ -0,0 +1,129 @@
+let rover_heading = 0
+let rover_x = 0
+let rover_y = 0
+let zoom_value = 1
+let MAX_ZOOM = 2
+let MIN_ZOOM = 0.25
+let canvas_pos
+let keys = { //javascript keycodes
+    'r': 82,
+    'w': 87,
+    'a': 65,
+    's': 83,
+    'd': 68
+}
+
+let base_lat
+let base_long
+$(document).ready(() => {
+    // the elements might not be initialized yet, must test
+    base_lat = $('#antenna-latitude').text()
+    base_long = $('#antenna-longitude').text()
+})
+
+function setup() {
+    let canvas = createCanvas(450, 300)
+    canvas.parent('#canvasContainer')
+    canvas_pos = $('#defaultCanvas0').offset()
+    angleMode(DEGREES)
+}
+
+function draw() {
```

Comment:
I don't understand what this function does, please give a comment explaining it and potentially give it a more descriptive name. Are you changing the orientation and position of the rover on the map? That's not obvious from the name.

Reply:
I was referring to the draw() function lol

Reply:
it's a p5.js function that runs on a loop